Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -9,7 +9,14 @@ class DocumentViewModel: ObservableObject {
     private(set) var annotations: [AnnotationViewModel]
     private(set) var pdfDocument: PdfViewModel
 
+    /*
+     I added this because I think eventually we will need it. But I make it
+     default empty array first because otherwise we can't compile yet
+     */
+    private(set) var selectionBoxes: [SelectionBoxViewModel] = []
```

Comment:
Hmm, why should we associate the selection boxes with the document? Except the selection box that is currently being created, each selection box should be encapsulated within its corresponding annotation. 

Or is this just temporary?

Reply:
Alrightss, I took it out
I don't forsee that we need it at all, it should be in the corresponding annotation that's right
I saw your other comments too. I'll work on them as well